test/Microsoft.TestPlatform.AcceptanceTests/Extension/RunnerCompatibilityDataSource.cs

```diff
@@ -75,7 +76,9 @@ public override void CreateData(MethodInfo methodInfo)
         _builder.DebugDataCollector = DebugDataCollector;
         _builder.DebugVSTestConsole = DebugVSTestConsole;
         _builder.DebugTestHost = DebugTestHost;
-        _builder.NoDefaultBreakpoints = NoDefaultBreakpoints;
+        _builder.DebugStopAtEntrypoint = DebugStopAtEntrypoint;
+
+        _builder.JustRow = JustRow < 0 ? null : JustRow;
```

Comment:
Is it worth checking for upper bound?

Reply:
It throws and fails or something. It's a utility, I will continue making it better. But outside of this PR :)